docs/src/main/paradox/common/timeouts.md

```diff
@@ -80,3 +80,10 @@ Tweaking it should rarely be required, but it allows erroring out the connection
 is unable to be established for a given amount of time.
 
 it can be configured using the `akka.http.client.connecting-timeout` setting.
+
+### Connection Keep Alive timeout
```

Comment:
"keep alive" sounds like "extend the life of a connection that would otherwise be closed", while this feature is actually "reduce the life of a connection that would otherwise remain open".

Perhaps we should consider renaming it to "Max connection age" of "Max connection lifetime" (in the setting name as well)? WDYT?

Reply:
Also the "timeout" word does not makes any sense here, so yes lets rename it. (I will have time tomorrow)
max-connection-lifetime sounds fancier :+1: